contracts/crowdsale/SimpleCrowdsaleBase.sol

```diff
@@ -114,6 +116,11 @@ contract SimpleCrowdsaleBase is ArgumentsChecker, ReentrancyGuard, IInvestmentsW
         return true;
     }
 
+    /// @notice whether to apply hard cap check logic via getMaximumFunds() method 
+    function hasHardCap() constant internal returns (bool) {
+        return true;
```

Comment:
Maybe `getMaximumFunds() != 0` or `getMaximumFunds() != uint(-1)`? It'll be good but overridable default I think.